cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -184,6 +184,30 @@ spec:
         # kubelet directory path, if kubelet configured to use other than /var/lib/kubelet path.
         #- name: ROOK_CSI_KUBELET_DIR_PATH
         #  value: "/var/lib/kubelet"
+        # (Optional) Ceph Provisioner NodeAffinity.
```

Comment:
How about adding a comment that shows typical usage? For example, the provisioner would be best to start on the same nodes as other ceph daemons.